pyproject.toml

```diff
@@ -46,6 +46,7 @@ django-ansible-base = { git = "https://github.com/ansible/django-ansible-base.gi
     "channel_auth",
     "rbac",
 ] }
+django-redis = "5.4.*"
```

Comment:
Instead of pulling this in directly, I'd modify the ansible-base to include the extra `redis_client` and that should pull in django-redis and anything required in the future.